firmware/user/modes/mode_joust_game.c

```diff
@@ -1167,8 +1167,8 @@ void ICACHE_FLASH_ATTR joustUpdateDisplay(void)
         (OLED_HEIGHT / 2 + 18) + 5,
         WHITE);
 
-    // Find the difference from the rolling average, scale it to 118px (5px margin on each side)
-    int16_t diffFromAvg = ((joust.mov - joust.rolling_average) * 118) / 43;
+    // Find the difference from the rolling average, scale it to 220px (5px margin on each side)
+    int16_t diffFromAvg = ((joust.mov - joust.rolling_average) * 220) / 43;
```

Comment:
@adangert why did you go with 220? The OLED is only 128px wide and only has 118px for the bar to be drawn in.

Reply:
No matter how much I shook the swadge, I could only get the bar to go to half, or the game to end, so it was just with some user testing to make the bar look better.

Reply:
alright!